cmd/easeprobe/main.go

```diff
@@ -54,6 +56,24 @@ func main() {
 		os.Exit(-1)
 	}
 
+	// Create the pid file
+	d, err := daemon.NewPIDFile(c.Settings.PIDFile)
+	if err != nil {
+		log.Errorf("Error: Cannot create the PID file: %s", err)
+		// Try Working Directory
+		pidfile := filepath.Join(global.GetWorkDir(), global.DefaultPIDFile)
```

Comment:
if i understand this correctly, when we fail to create the pid file defined by the settings, we try to create a pid file on the current directory? 

If so this is kind of counter intuitive. I think that the proper action when we fail to create a pid is to not start.

Reply:
Yes, your understanding is right!

And thanks for this review, I will correct the behavior as you mentioned - cannot create the pid file, just failed to start.